typescript/cli/src/context/context.ts

```diff
@@ -64,6 +68,36 @@ export async function signerMiddleware(argv: Record<string, any>) {
    */
   const chains = await chainStrategy.resolveChains(argv);
 
+  /**
+   * Load and create AltVM Providers
+   */
+  const altVmChains = chains.filter(
```

Comment:
Why are the protocol providers created here and not in the `contextMiddleware` function?